pkg/plugins/platform/local_test.go

```diff
@@ -557,12 +636,17 @@ func TestRouterNotRunning(t *testing.T) {
 	}
 }
 
-// constructContainerName builds a container name given the type (web/db/dba) and the app
-func constructContainerName(containerType string, app platform.App) (string, error) {
+// getContainerByType builds a container name given the type (web/db/dba) and the app
+func getContainerByType(containerType string, app platform.App) (string, error) {
 	container, err := app.FindContainerByType(containerType)
 	if err != nil {
 		return "", err
 	}
 	name := dockerutil.ContainerName(container)
 	return name, nil
 }
+
+func removeAllErrCheck(path string, assert *asrt.Assertions) {
+	err := os.RemoveAll(path)
+	assert.NoError(err)
+}
```

Comment:
It looks like you had some trouble with os.RemoveAll() - Was that on windows testing on the test we had to move out? This code doesn't seem like it should be that necessary, although I don't think it does any harm.

Reply:
This is the code that Brad and I use to cleanup the directory name changes that we make to mock a missing directory in TestLocalStopMissingDirectory, TestDescribeMissingDirectory, and CleanupWithoutCompose.